routers/install/install.go

```diff
@@ -147,8 +149,19 @@ func Install(ctx *context.Context) {
 
 	// Server and other services settings
 	form.OfflineMode = setting.OfflineMode
-	form.DisableGravatar = setting.DisableGravatar
-	form.EnableFederatedAvatar = setting.EnableFederatedAvatar
+	disableGravatarSetting, _ := system_model.GetSetting(system_model.KeyPictureDisableGravatar)
+	if disableGravatarSetting != nil {
+		form.DisableGravatar = disableGravatarSetting.GetValueBool()
+	} else {
+		form.DisableGravatar = false
+	}
+
+	enableFederatedAvatarSetting, _ := system_model.GetSetting("")
```

Comment:
key missing ?

Reply:
@lunny shouldn't this contain "picture.enable_federated_avatar" ?

Reply:
Good catch, fixed.